### PR TITLE
default reverse order

### PR DESCRIPTION
**Pull request by rzulian**
simple implementation of the reverse order that you can need in some phase of your game

### CODE REVIEW

**Comment by rzulian** [comment]
I signed it!

File: src/core/turn-order.js [hunk 1]
```diff
@@ -48,6 +48,22 @@ export const TurnOrder = {
...
   },
 
+    /**
+   * DEFAULT-REVERSE
+   *
+   * The default round-robin turn order starting from the last player.
+   */
+  DEFAULT_REVERSE: {
+    first: (G, ctx) => ctx.numPlayers -1 + '',
+    next: (G, ctx) => {
```
Comment (nicolodavis) [inline src/core/turn-order.js]: This should also work:
(+ctx.currentPlayer + ctx.numPlayers - 1) % ctx.numPlayers + ''

**Comment by nicolodavis** [comment]
Also, you should use the label 'ptal', not add it to your Pull Request description.

File: src/core/turn-order.js (hunk 1 above)
Comment (rzulian) [reply src/core/turn-order.js]: yes, this is cleaner

**Comment by rzulian** [comment]
Sorry for the label. This is my first pull request ever!

**Comment by nicolodavis** [comment]
Yeah, I realized that just now. Removing the recommendation to add labels from the guidelines.

Also, looks like you have a lint error: https://travis-ci.org/google/boardgame.io/builds/346812962?utm_source=github_status&utm_medium=notification

File: src/core/turn-order.js [hunk 2]
```diff
@@ -48,6 +48,16 @@ export const TurnOrder = {
     next: (G, ctx) => (+ctx.currentPlayer + 1) % ctx.numPlayers + '',
   },
 
+    /**
+   * DEFAULT-REVERSE
+   *
+   * The default round-robin turn order starting from the last player.
+   */
+  DEFAULT_REVERSE: {
```
Comment (nicolodavis) [inline src/core/turn-order.js]: Rename this to just `REVERSE`.

**Comment by rzulian** [comment]
@nicolodavis I thinking to extend the TurnOrder to manage custom order, which is very common in boardgames. Problem is that current implementation is assuming player order starting from currentPlayer='0'.  What if I need to start a new phase or round base on the pass order of the previous phase? 
What do you think?

**Comment by nicolodavis** [comment]
Yes, you can use `G` to store the starting player and use a custom turn order to read that field from `G`.

File: package.json [hunk 3]
```diff
@@ -72,7 +72,7 @@
     "babel-watch": "^2.0.7",
     "chess.js": "^0.10.2",
     "coveralls": "^3.0.0",
-    "cross-env": "^5.1.1",
+    "cross-env": "^5.1.3",
```
Comment (nicolodavis) [inline package.json]: Please revert this and the change to package-lock.json.

**Comment by nicolodavis** [comment]
I'm assuming that you want to close this in favor of https://github.com/google/boardgame.io/pull/130.

**Pull request closed without merging**